go.mod

```diff
@@ -15,55 +15,46 @@ require (
 	github.com/go-openapi/swag v0.19.15
 	github.com/go-openapi/validate v0.20.1
 	github.com/go-sql-driver/mysql v1.6.0
-	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/golang/glog v1.0.0
 	github.com/golang/protobuf v1.5.2
 	github.com/google/addlicense v0.0.0-20200906110928-a0294312aa76
-	github.com/google/go-cmp v0.5.6
+	github.com/google/go-cmp v0.5.7
 	github.com/google/uuid v1.3.0
 	github.com/gopherjs/gopherjs v0.0.0-20200217142428-fce0ec30dd00 // indirect
 	github.com/gorilla/mux v1.8.0
 	github.com/grpc-ecosystem/grpc-gateway v1.16.0
 	github.com/jinzhu/gorm v1.9.12
 	github.com/kubeflow/pipelines/api v0.0.0-20211026071850-2e3fb5efff56
 	github.com/lestrrat-go/strftime v1.0.4
-	github.com/lib/pq v1.9.0 // indirect
 	github.com/mattn/go-sqlite3 v2.0.1+incompatible
 	github.com/minio/minio-go v6.0.14+incompatible
 	github.com/peterhellberg/duration v0.0.0-20191119133758-ec6baeebcd10
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.11.0
+	github.com/prometheus/client_golang v1.11.1
 	github.com/robfig/cron v1.2.0
 	github.com/sirupsen/logrus v1.8.1
 	github.com/smartystreets/assertions v0.0.0-20190401211740-f487f9de1cd3 // indirect
-	github.com/spf13/viper v1.8.1
+	github.com/spf13/viper v1.10.0
 	github.com/stretchr/testify v1.7.0
-	github.com/tektoncd/pipeline v0.31.4
+	github.com/tektoncd/pipeline v0.35.0
 	github.com/tidwall/pretty v1.1.0 // indirect
 	go.uber.org/zap v1.19.1
-	golang.org/x/net v0.0.0-20211015210444-4f30a5c0130f
-	google.golang.org/genproto v0.0.0-20211016002631-37fc39342514
-	google.golang.org/grpc v1.41.0
+	golang.org/x/net v0.0.0-20220225172249-27dd8689420f
+	google.golang.org/genproto v0.0.0-20220303160752-862486edd9cc
+	google.golang.org/grpc v1.44.0
 	google.golang.org/grpc/cmd/protoc-gen-go-grpc v1.1.0
 	google.golang.org/protobuf v1.27.1
-	k8s.io/api v0.21.5
-	k8s.io/apimachinery v0.21.5
-	k8s.io/client-go v0.21.5
-	k8s.io/code-generator v0.21.5
-	sigs.k8s.io/controller-runtime v0.9.7
+	k8s.io/api v0.23.5
```

Comment:
good question. I didn't try to lever the version. that version comes from here: https://github.com/tektoncd/pipeline/blob/main/go.mod#L24

Reply:
do you want me to try a specific version lower than `0.23.5`?

Reply:
It should be fine, but usually having a higher api version means it might not able to run on lower k8s version. In this case we will need to update the docs stating that we at least need k8s 1.21+.

Reply:
will update that in the `kfp_tekton_install.md`